tools/tpm2_dump_capability.c

```diff
@@ -562,6 +612,10 @@ static int dump_tpm_capability (TPMU_CAPABILITIES *capabilities) {
         dump_algorithms (capabilities->algorithms.algProperties,
                          capabilities->algorithms.count);
         break;
+    case TPM_CAP_ECC_CURVES:
+	dump_ecc_curves (capabilities->eccCurves.eccCurves,
+                         capabilities->eccCurves.count);
+	break;
```

Comment:
I think it would be a good idea to keep the capabilities sorted according to 6.12 TPM_CAP (Capabilities) table in Trusted Platform Module Library part 2.

Rest of the patch looks good, thank a lot for proposing it.

Reply:
Sorry I don't quite get it. The previous sorting [PROPERTIES, ALGS, COMMANDS] didn't correspond 6.12 as well.

Reply:
Indeed and that's not your fault, but I meant that we should try to keep it sorted from now on (since is quite likely that more properties will be added the future).

We can later move TPM_CAP_TPM_PROPERTIES to the end to match the table order.

Reply:
OK, this makes sense.